docs/guides/specify-runtime-options.mdx

```diff
@@ -333,26 +218,14 @@ estimator = Estimator(mode=backend)
 estimator.options.max_execution_time = 2500
 ```
 
-  </TabItem>
-
-  <TabItem value="PrimV1" label="V1 primitives">
-```python
-options = Options()
-estimator.options.max_execution_time = 2500
-
-estimator = Estimator(backend=backend)
-```
-</TabItem>
-</Tabs>
-
 <span id="no-error-mitigation"></span>
 ## Turn off all error mitigation and error suppression
 
 You can turn off all error mitigation and suppression if you are, for example, doing research on your own mitigation techniques. To accomplish this, for EstimatorV2, set `resilience_level = 0`. For SamplerV2, no changes are necessary because no error mitigation or suppression options are enabled by default.
 
 Example:
 
-Turn off all error mitigation and suppression in EstimatorV2.
+Turn off all error mitigation and suppression in Estimator.
 
 ```python
 from qiskit_ibm_runtime import EstimatorV2 as Estimator, Options, QiskitRuntimeService
```

Comment:
This should not import `Options` since it's only used for V1 and will be removed.